### DISCUSSION TITLE
Django like superadmin for Remix

### DISCUSSION THREAD

**Discussion by Rafi993** [remix-run/remix#8449] [Proposals]
Super admin in Django is really helpful for quick prototyping. You could use something like Prisma studio now but the problem is it doesn't restrict access to specific fields and it is not as smooth as Django's superadmin. It would be nice if Remix had something similar built-in or at least like a plugin that one can opt-in for.

**Comment by sergiodxa** [comment]
Things like Django super admin works on Django ecosystem because you know all Django apps typically use the same ORM (the default that comes with Django).

But Remix doesn't prescribe an ORM, that's up to you, you could even not use an ORM at all (e.g. if you fetch an API), this makes building an admin way more complex because you can't expect Prisma to be used all the time, and if you do you're limiting it Remix apps that use Prisma.

There are libraries like [React Admin that has Remix support](https://marmelab.com/react-admin/Remix.html) so you could use that to integrate an admin into your Remix app, and they have already worked on adapters for different possible DB and APIs.
